app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class SessionsController < ApplicationController
+  # require login needs to be created in the ApplicationController to create flash message
+  # skip_before_action :require_login, only: [:new, :create]
+  before_action :redirect_if_logged_in, only: [:new, :create]
+  def new
+
+  end
+
+  def create
```

Comment:
It may be nitpicky now, but you really want to pay attention to indentation and follow the rules of style guides. It will make your code easier to read and you won't have to spend as much time trying to comply with 'style guide conventions' if you just get into the habits now.